packages/discovery-provider/plugins/pedalboard/apps/trending-challenge-rewards/src/trending.ts

```diff
@@ -99,6 +99,7 @@ export const queryTopFiveTrending = async (
 }
 
 export const queryHandles = async (
```

Comment:
This is exported. Do we call it outside this module? And if so, is adding the new argument at the beginning going to break those locations?

Reply:
good eyes. it's exported but no one else is using atm